docs/docs/reference/erased-terms.md

```diff
@@ -177,12 +177,11 @@ class K(erased x: Int) { ... }
    * `(implicit erased T1, T2) => R  <:<  (erased T1, T2) => R`
    *  ...
 
-Note that there is no subtype relation between `erased T => R` and `T => R` (or `implicit erased T => R` and `implicit T => R`)
+   Note that there is no subtype relation between `erased T => R` and `T => R` (or `implicit erased T => R` and `implicit T => R`)
 
 
 4. Eta expansion
-
-if `def f(erased x: T): U` then `f: (erased T) => U`.
+   if `def f(erased x: T): U` then `f: (erased T) => U`.
```

Comment:
Will rendered on the same line as the previous line. So we might as well rewrite it as:
```
4. Eta expansion: if `def f(erased x: T): U` then `f: (erased T) => U`.
```

Reply:
Was not my intention, I'll revert this particular change